litho-core/src/main/java/com/facebook/litho/config/ComponentsConfiguration.java

```diff
@@ -145,6 +145,9 @@
    */
   public static boolean enableShouldCreateLayoutWithNewSizeSpec = true;
 
+  /** Ignore duplicate transition keys in the same layout, rather than throwing an exception. */
+  public static boolean ignoreDuplicateTransitionKeysInLayout = true;
```

Comment:
Do we expect some clients to configure this to false? Should this just be the new logic? As a static, it also seems awkward because it's hard to know if it get's changed by another view integrating Litho in the same app.

Reply:
I will defer this question to the Litho team. Perhaps the default configuration item should be false to preserve the existing logic and we can change it to true for our use cases.

Reply:
Yeah, I believe we should probably set it `false`.

Reply:
Sounds good, I have turned this by default to false to preserve the current logic.